app/models/customer.rb

```diff
@@ -9,6 +9,18 @@ class Customer < ActiveRecord::Base
 
   has_one :profile
 
+
+  ## Messages sent to me
+  # 1)SQL:
+  # SELECT * FROM messages where messages.receiver_id=#{self.id}
+  # 2) Create models for 
+  has_many :received_messages, class_name: 'Message', foreign_key: :receiver_id
```

Comment:
"I want to find all of the records in the messages table where my customer id matches the messages receiver_id"... does that sound right?